Leanplum-SDK/Classes/Utils.m

```diff
@@ -2,25 +2,8 @@
 //  Utils.m
 //  Leanplum
 //
-//  Created by Ben Marten on 6/6/16.
-//  Copyright (c) 2016 Leanplum, Inc. All rights reserved.
+//  Copyright (c) 2018 Leanplum, Inc. All rights reserved.
 //
-//  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why we need to remove this from now?

Reply:
helps clean the files since we don't have a need for it

![image](https://user-images.githubusercontent.com/761132/39909888-9810c5ae-54a9-11e8-9c8d-f62818585d2a.png)

Reply:
That was not for open source code!

Reply:
i only removed these for the files that we haven't open sourced... Leanplum.h and LPInbox.h retain the licenses

Reply:
Whole Leanplum-iOS-SDK repo is open sourced..